config/config.go

```diff
@@ -499,6 +500,15 @@ type StorageConfig struct {
 	ExemplarsConfig *ExemplarsConfig `yaml:"exemplars,omitempty"`
 }
 
+// TracingConfig configures the tracing options.
+type TracingConfig struct {
+	GrpcClient       bool    `yaml:"grpc_client,omitempty"`
```

Comment:
We should preserve the capitalization of "[g]RPC".

```suggestion
	GRPCClient       bool    `yaml:"grpc_client,omitempty"`
```